README.md

```diff
@@ -1 +1,23 @@
-# Newyork
\ No newline at end of file
+For this project we had to create a clone and copy the HTML AND CSS Elements from
+https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0
+
+we used knowledge from content on flex boxs and grids and other positioning and layout options.
+
+instructions used for project build
+
+1 Create a new HTML document.
+2 Think about all the elements on the page and how they are grouped together.
+Lay out the basic structure of the page using empty semantic HTML elements that are appropriately sized and positioned. You may want to make their background colors different so you can see them. Don’t forget to use your browser’s developer tools (right click on the page, click “Inspect Element”)!
+3 Set up the top navigation bar (again, don’t worry about making it change when you scroll down the page). How can you get it to stick to the top of the page as you scroll down?
+Fill in and position the additional sections. Don’t worry about enclosing every paragraph of text in <p> tags.
+4 Push your solution to GitHub.
+
+Link to my live site here -->
+https://stephanie041996.github.io/Newyork/
+
+![image](https://user-images.githubusercontent.com/70282171/108333735-f0febb80-71d9-11eb-881b-54c5e9f4427e.png)
+
+This project was built using HTML and CSS
+
+Authors name :Stephanie Sakuhuni
+Linkedin :https://www.linkedin.com/in/stephanie-sakuhuni-a81029140/
```

Comment:
## Improve the README
- [ ] The `README` looks okay but it can be bettered still. Kindly use this [template](https://github.com/microverseinc/readme-template) as an inspiration on areas you might want to improve upon and how to arrange items on this document.